package Java8Streams.Java8Coding.ProductCustomer;

public class Customer {

    private Long id;
    private String name;
    private Integer tier;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Customer(Long id, String name, Integer tier) {
        this.id = id;
        this.name = name;
        this.tier = tier;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setTier(Integer tier) {
        this.tier = tier;
    }

    public String getName() {
        return name;
    }

    public Integer getTier() {
        return tier;
    }


    public Object getProductCount() {
        return 1;
    }
}
